frontend/app/.server/environment/features.ts

```diff
@@ -1,16 +1,17 @@
-import { z } from 'zod';
+import * as v from 'valibot';
 
-import * as ValidationUtils from '~/utils/validation-utils';
+import { stringToBooleanSchema } from '../validation/string-to-boolean-schema';
 
-export type Features = Readonly<z.infer<typeof features>>;
+import { boolToString } from '~/utils/boolean-utils';
+
+export type Features = Readonly<v.InferOutput<typeof features>>;
+
+const isProduction = process.env.NODE_ENV === 'production';
 
 export const defaults = {
-  ENABLE_DEVMODE_OIDC:
-    process.env.NODE_ENV === 'production' //
-      ? 'false'
-      : 'true',
+  ENABLE_DEVMODE_OIDC: isProduction ? boolToString(false) : boolToString(true),
```

Comment:
```suggestion
  ENABLE_DEVMODE_OIDC: isProduction ? 'false' : 'true',
```

Why make an extra function call to do what can just be done inline?

Reply:
Changed.